scripts/repl.js

```diff
@@ -31,8 +31,10 @@
   var getQuery = function() {
     var query = window.location.hash.replace(/^\#\?/, '');
     return _.transform(query ? query.split('&') : null, function(result, val) {
-      val = val.split('=');
-      result[val[0]] = decodeURIComponent('' + val[1]);
+      var equalsIndex = val.indexOf('=');
+      var name = val.substring(0, equalsIndex);
+      var value = val.substring(equalsIndex + 2);
```

Comment:
@thejameskyle Using the fix from @fabiomcosta , I created a fix branch on the refactor from #73 at d481f0ac020ffbf8f82f3c30c01d200841a201f1